src/main/resources/assets/armorplus/lang/zh_cn.lang

```diff
@@ -247,12 +243,14 @@ item.armorplus.dev_leggings.name=开发者护腿
 item.armorplus.dev_boots.name=开发者靴子
 item.armorplus.dev_tool.name=开发者工具
 item.armorplus.dev_item.name=NBT物品
-#item.armorplus.the_dragon_team.name=TheDragonTeam
-#item.armorplus.modded_city.name=ModdedCity
-#item.armorplus.jon_bams.name=JonBams
-#item.armorplus.beam.name=Beam
-#item.armorplus.twitch.name=Twitch
+item.armorplus.the_dragon_team.name=TheDragonTeam
```

Comment:
Remain commented; I repeat, remain commented; this applies to any untranslated entries here